libs/frontend/ui-build/src/components/component-menu/ComponentDimensions.vue

```diff
@@ -135,6 +135,22 @@ const toggleCollapse = () => {
   )
 }
 
+const updateStyle = (property: Css, value: string | undefined) => {
```

Comment:
This is to fix an issue where jumping to the next size input by pressing Tab would directly update the style of the selected component.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/8262f6df-808b-43ad-898f-12ab216c779a